jest.config.ts

```diff
@@ -3,6 +3,7 @@ module.exports = {
   transform: {
     "^.+\\.[t|j]sx?$": "babel-jest",
   },
+  transformIgnorePatterns: ["node_modules/(?!(@runcitadel))/"],
```

Comment:
do we still need this?

Reply:
I believe so, yes

Reply:
wasn't there a change in that package? 
(and I don't like such special cases :) )

Reply:
Not sure, will check

Reply:
My understanding is that runcitadel's dist code is coming as ESM (using import/export) which can't be handled by Jest. Therefor we need to transform the citadel code eventough normally we do not compile any code from `node_modules`.  
@dylancom what do you think about this?

Reply:
ah I see. thanks for the explanation. 

but then we can merge this PR?

Reply:
I think so, yes